webapp/channels/src/packages/mattermost-redux/src/actions/channels.ts

```diff
@@ -293,7 +293,7 @@ export function convertGroupMessageToPrivateChannel(channelID: string, teamID: s
             return {};
         }
 
-        return updatedChannel;
+        return response;
```

Comment:
Should we be sending back response.data? because i am guessing that contains the `Channel` data

Reply:
So everywhere we return API call data, we return the whole response as it contains stuff like `error`, `data`, etc.

Reply:
Ok but the return type says only of that of data isn't it ?

Reply:
circling back on this one again the type of response here is 
```
response: Response;
    headers: Map<string, string>;
    data: T;
```

So in this line we should be returning response.data instead since this `convertGroupMessageToPrivateChannel` action expects Channel as return type